scripts/zones/Chateau_dOraguille/npcs/Halver.lua

```diff
@@ -187,24 +191,23 @@ function onEventFinish(player,csid,option)
 	elseif(csid == 0x0016) then
 		player:setVar("MissionStatus",4);
 	elseif(csid == 0x0009) then 
-	if(player:getFreeSlotsCount() == 0) then 
-	player:messageSpecial(ITEM_CANNOT_BE_OBTAINED,181);
-	player:setVar("Flagsando",1);
-	else	player:addItem(181);
-	player:messageSpecial(ITEM_OBTAINED,181);
-	end
-	player:setVar("SANDO92",0);
-	player:completeMission(SANDORIA,THE_HEIR_TO_THE_LIGHT);
-	player:setRank(10);
-	player:addGil(100000);
-	player:messageSpecial(GIL_OBTAINED,100000);
-	player:setTitle(295);
-	player:setVar("SandoEpilogue",1);
+			if(player:getFreeSlotsCount() == 0) then 
+				player:messageSpecial(ITEM_CANNOT_BE_OBTAINED,181);
+				player:setVar("Flagsando",1);
+			else	player:addItem(181);
+				player:messageSpecial(ITEM_OBTAINED,181);
+			end
+		player:setVar("MissionStatus",0);
+		player:completeMission(0, 23);

```

Comment:
Oh theres another.
Don't change them from named global vars as defined in missions.lua to being just number please.